forum/search/search_manager.py

```diff
@@ -73,6 +73,17 @@ def get_suggested_text(
         response: dict[str, Any] = self.client.search(
             index=self.index_names, body=suggest_body
         )
+        suggestion = self._extract_suggestion(
+            response, [f"{field}_suggestions" for field in suggestion_fields]
+        )
+
+        # TODO: Remove the extra call for searching the text in the comments.
```

Comment:
I don't understand. Do we still need this?

Reply:
Yes, we didn't found the solution of this yet.